pkgs/development/python-modules/antlr4_9-python3-runtime/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildPythonPackage, isPy3k, python
+, antlr4_9
+}:
+
+buildPythonPackage rec {
+  pname = "antlr4_9-python3-runtime";
+  inherit (antlr4_9.runtime.cpp) version src;
+  disabled = !isPy3k;
```

Comment:
Should probably be changed to `pythonOlder "3.5"` since it won't build without `typing` package otherwise (although surely no-one would notice this difference).